dragonfly/implementations/PsyllidProvider.py

```diff
@@ -0,0 +1,404 @@
+'''
```

Comment:
naming convention: use snake_case (psyllid_provider.py), the class names can remain CamelCase